### DISCUSSION TITLE
Ikea light

### DISCUSSION THREAD

**Discussion by jamvandermeer** [isaacrlevin/presencelight#385] [Ideas]
is it possible to control Ikea lights? they are much cheaper than Philips Hue :-)

https://learn.pimoroni.com/tutorial/sandyj/controlling-ikea-tradfri-lights-from-your-pi

**Comment by isaacrlevin** [comment]
I don't own any of these but would be more than happy to review a pull request for something like that

**Comment by isaacrlevin** [comment]
Closing
